### DISCUSSION TITLE
0 frames predicted when adding a second video

### DISCUSSION THREAD

**Discussion by IsabelleSajonia** [talmolab/sleap#1475] [Help!]
Hi. I was able to successfully run inference on the original video I was training on and exported this as a HDF5 file. I wanted to use this same model (in the same project file) to train on a second video that is slightly different, and eventually run inference on this second video. After a few training sessions my predictions looked good, so I tried to run inference but got the message that 0 frames were predicted. It is no longer outputting predictions on the original video either. The height and width, and number of channels are the same for the two videos. One is about 7,000 frames and the other is 8,700. I'm doing this through the GUI on Windows. I'm not sure if it just needs more training or if I need to just create a different model to make predictions on this new video. Thanks!

**Comment by talmo** [comment]
Hi @IsabelleSajonia,

Thanks for reaching out! Apologies for the delay -- we've had a bit of a hectic week.

If the model was working previously, it should continue to work (especially on the first video it sounds like).

Are you getting any other errors in the terminal?

Do you have previous versions of either the labels or the models that you can revert back to and trace back to what might be introducing the problem?

Some screenshots of the predictions (or lack thereof), the first vs second video, and the model configuration would all be super helpful for diagnosing what's going on as well.

Let us know when you get a chance.

Cheers,

Talmo
